libdnf/base/log_event.cpp

```diff
@@ -94,13 +114,17 @@ std::string LogEvent::to_string(
         case GoalProblem::NOT_AVAILABLE:
             return ret.append(utils::sformat(_("Packages for argument '{}' installed, but not available."), *spec));
         case GoalProblem::NO_PROBLEM:
```

Comment:
`GoalProblem::NO_PROBLEM` is now the same as `GoalProblem::ALREADY_INSTALLED`. That wasn't intended I suppose.

Reply:
Fixed